apps/common-app/src/apps/reanimated/examples/RuntimeTests/ReJest/TestRunner/WindowDimensionsMocker.ts

```diff
@@ -1,8 +1,8 @@
 import type {
   LayoutAnimation,
   LayoutAnimationStartFunction,
+  LayoutAnimationsValues,
```

Comment:
We can rename that for the sake of consistency
```suggestion
  LayoutAnimationValues,
```